src/main/java/com/javabom/baseballgame/domain/BaseballNumbers.java

```diff
@@ -0,0 +1,50 @@
+package com.javabom.baseballgame.domain;
+
+import java.util.Collections;
+import java.util.Set;
+
+import static com.javabom.baseballgame.domain.GameOutput.*;
+
+public class BaseballNumbers {
+    private static final int COUNT = 3;
+    private final Set<OrderedBaseballNumber> values;
+
+    public BaseballNumbers(final Set<OrderedBaseballNumber> values) {
+        this.values = values;
+        if (values.size() != COUNT) {
```

Comment:
```suggestion
        this.values = values;
        if (this.values.size() != COUNT) {
```
`this` 써주는게 안전하지 않을까?